CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ different versioning scheme, following the Haskell community's
   accepting `Comparer` for associative containers.
 * C++ codegen now can generate copy and move constructors with an allocator argument
   when a custom allocator is used and `--alloc-ctors` is passed to `gbc`.
+* C++ codegen now can use [`std::scoped_allocator_adaptor`](http://en.cppreference.com/w/cpp/memory/scoped_allocator_adaptor)
+  for strings and containers when custom allocator is used and `--scoped-alloc` flag
```

Comment:
Are sub-structs intentionally excluded?

Reply:
Discussed offline.

Reply:
To summarize the discussion: the sub-structs will get the top-level allocator and will use that to create a `std::scoped_allocator_adapter` around the top-level allocator for their own fields. The intent of this functionality is to make things like `someStruct.collectionField.emplace_back(arg1, arg2)` transparently use the allocator, less so to use the levels functionality of `std::scope_allocator_adapter`.